cmd/testReg/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+
+	"github.com/openshift/ci-tools/pkg/load"
+	"github.com/openshift/ci-tools/pkg/registry"
+)
+
+func main() {
```

Comment:
If we want to land this in the current PR we should use the normal workflow (`type options struct` etc) and be a little more rigorous, like adding help text to flags, etc